go.mod

```diff
@@ -54,7 +54,7 @@ require (
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
 	github.com/tendermint/go-amino v0.16.0
-	github.com/tendermint/tendermint v0.35.8
```

Comment:
`go mod tidy` did that -> https://github.com/cosmos/cosmos-sdk/runs/7445987115?check_suite_focus=true